code/datums/jobs/job/shipside.dm

```diff
@@ -1104,7 +1104,7 @@ You are also an expert when it comes to medication and treatment. If you do not
 //Researcher
 /datum/job/terragov/medical/researcher
 	title = MEDICAL_RESEARCHER
-	comm_title = "Rsr"
+	comm_title = "Researcher"
```

Comment:
?

Reply:
Though that it would be a little more obvious to understand what it means as "Rsr" or "Syn" in the icon file

Reply:
comm title is used as their title for comms by default.

Reply:
Oh